sdk/azcore/internal/pollers/util_test.go

```diff
@@ -22,8 +22,10 @@ import (
 func TestIsTerminalState(t *testing.T) {
 	require.False(t, IsTerminalState("Updating"), "Updating is not a terminal state")
 	require.True(t, IsTerminalState("Succeeded"), "Succeeded is a terminal state")
+	require.True(t, IsTerminalState("Completed"), "Completed is a terminal state")
 	require.True(t, IsTerminalState("failed"), "failed is a terminal state")
 	require.True(t, IsTerminalState("canceled"), "canceled is a terminal state")
+	require.True(t, IsTerminalState("cancelled"), "cancelled is a terminal state")
```

Comment:
Should we also test "Cancelled"?

Reply:
We're supposed to, and do at present, perform a case-insensitive comparison.  Perhaps instead of testing both permutations, what if we used input values that don't match either permutation (e.g. `cancELLed`)?

Reply:
Sounds good, I like it 👍

Reply:
Fixed